api/app/src/external/commonwell/patient.ts

```diff
@@ -0,0 +1,288 @@
+import {
+  AddressUseCodes,
+  CommonWell,
+  getId,
+  getIdTrailingSlash,
+  Identifier,
+  IdentifierUseCodes,
+  NameUseCodes,
+  Patient as CommonwellPatient,
+  Person as CommonwellPerson,
+  RequestMetadata,
+} from "@metriport/commonwell-sdk";
+import { ExternalMedicalPartners } from "..";
+import { getPatientWithDependencies } from "../../command/medical/patient/get-patient";
+import { setCommonwellId } from "../../command/medical/patient/update-patient";
+import { Facility } from "../../models/medical/facility";
+import { Organization } from "../../models/medical/organization";
+import {
+  Gender,
+  generalTypes,
+  Patient,
+  PatientData,
+  PatientDataExternal,
+} from "../../models/medical/patient";
+import { driversLicenseURIs, medicareURI, oid, passportURI, ssnURI } from "../../shared/oid";
+import { Util } from "../../shared/util";
+import { makeCommonWellAPI, organizationQueryMeta } from "./api";
+
+const genderMapping: { [k in Gender]: string } = {
+  F: "F",
+  M: "M",
+  O: "UN",
+  U: "UN",
```

Comment:
We want to support FHIR, so I think this should be our common denominator. I'm assuming other networks/partners might follow FHIR, then we can provide the closest API to FHIR we can.

Why do you think it would create unintended demographics? I think as long as we always convert using the same logic, it should work fine.

Reply:
The scenario I see is a cx thinking "I haven't collected my patient's gender - oh well, I'll just set it to 'unknown' for now since the API supports this", and allowing for this could create more demographic conflicts in our future master patient index (MPI).

FHIR supports it, but doesn't mean we need to, as our MPI will be the source of truth. Thought here being the more randomness we allow in our API, the harder job we will have building up that MPI across customers.

It seems like if we restrict this to M/F we can always add O/U down the road if necessary by simply commenting the code, but can't do things the other way around.

Reply:
We'll make this `genderAtBirth` and have only `M`|`F`.